modules/machine-health-checks-pausing.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+
+// * updating/updating-cluster-cli.adoc
+
+[id="machine-health-checks-pausing_{context}"]
+= Pausing a MachineHealthCheck resource
+
+During the upgrade process, nodes in the cluster might become temporarily unavailable. The `MachineHealthCheck` might identify such nodes as unhealthy and reboot them. To avoid rebooting such nodes, pause all the `MachineHealthCheck` resources before updating the cluster.
+
+.Prerequisites
+
+* Install the `oc` command line interface.
+
+.Procedure
+
+* To pause the machine health checks, add `cluster.x-k8s.io/paused=""` annotation to the `MachineHealthCheck` resource. To do so, run the following command:
++
+[source,terminal]
+----
+$ oc -n <namespace> annotate mhc <mhc-name> cluster.x-k8s.io/paused=""
```

Comment:
Is the user going to have the information for these variable on hand? At some point, it might be a good improvement to point out the default namespace name, `openshift-machine-api`, and how to get the mhc resource name.

Reply:
+1 to upgrade, not installation.

Reply:
I have now updated the commands and added a step to get the MHC resource name.